pipelines/appco/Jenkinsfile

```diff
@@ -10,34 +10,49 @@ def RUN_V2_TEST = params.RUN_V2_TEST ? params.RUN_V2_TEST : false
 def ENABLE_MTLS = params.ENABLE_MTLS ? params.ENABLE_MTLS : false
 def USE_HDD = params.USE_HDD ? params.USE_HDD : false
 def WORKSPACE = "/src/longhorn-tests"
+def LONGHORN_INSTALL_METHOD = AIR_GAP_INSTALLATION ? 'manifest' : 'helm'
```

Comment:
Can we know why we must install Longhorn by manifest when it's air-gap installation?

Reply:
AppCo air gap support helm installation not implement in this PR and have another [ticket](https://github.com/rancher/suse-storage/issues/32).